task_10/src/Footer/Footer.js

```diff
@@ -0,0 +1,28 @@
+require('./Footer.scss')
+const octicons = require("octicons")
+
+import { Component } from '../Framework'
+
+export default class Footer extends Component {
+	constructor() {
+		super();
+
+		this.host = document.createElement('div');
+		this.host.classList.add('footer__container');
+	}
+
+	render() {
+		return `
+<footer class="footer">
```

Comment:
I'm not sure, but class names `footer` for footer, or `head` for head, etc, it's a bad pattern. They pretty useless, because they are not descriptive. Also, you can use tag selector, if you really need to find footer.